cassandra_nodetool/tox.ini

```diff
@@ -0,0 +1,43 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    unit_test
+    integration
+    flake8
+
+[testenv]
+platform = linux|darwin|win32
+
+[testenv:unit_test]
+deps =
+    ../datadog_checks_base
```

Comment:
As @zippolyte mentioned we'll need to add https://github.com/DataDog/integrations-core/pull/1796/files#diff-5fdef06340079bd5037a38d5b9b97086R15.